rllib/evaluation/rollout_worker.py

```diff
@@ -860,14 +860,15 @@ def compute_gradients(
                 summarize(samples)))
         if isinstance(samples, MultiAgentBatch):
             grad_out, info_out = {}, {}
-            if self.tf_sess is not None:
-                builder = TFRunBuilder(self.tf_sess, "compute_gradients")
+            if self.policy_config.get("framework") == "tf":
```

Comment:
This was buggy code: Each policy nowadays has its own tf-graph (instead of the workers/trainer carrying the graph), so we have to loop through each policy here.